public/make_thread.php

```diff
@@ -1,27 +1,49 @@
+<?php
+
+require('init_form.php');
+
+session_start();
+
+if (formIsBeingSubmitted()) {
```

Comment:
Nope, if the form is not being submitted, we don't want to run any of this code. We just want to display the HTML below.